test/functional/ui/popupmenu_spec.lua

```diff
@@ -1854,8 +1854,56 @@ describe('builtin popupmenu', function()
       before_each(function()
         --row must > 10
         screen:try_resize(40, 11)
+        screen:set_default_attr_ids({
+          [1] = { foreground = Screen.colors.Blue1, bold = true },
```

Comment:
why is all of this needed? most tests use the default highlights

Reply:
I know. But there are many different color attr here than what was set at the beginning.

Reply:
I checked it out and it needs to start from 100. Many tests need to be changed. And the values ​​replace a few default ones 🤔

Reply:
Yeah this file still needs to get the general `add_extra_attr_ids()` treatment (mostly) @bfredl has been working on. I think the required changes are wanted (so remove all `set_default_attr_ids()` from the file and replace them with `set_extra_attr_ids()`), but could leave it as a TODO I guess.